code/create_app.py

```diff
@@ -448,4 +448,10 @@ def speech_config():
 
             return {"error": "Failed to get speech config"}, 500
 
+    @app.route("/api/getAssistantType", methods=["GET"])
+    def getAssistantType():
+        ConfigHelper.get_active_config_or_default.cache_clear()
```

Comment:
Oh yeah! I missed this line in Admin page _"Configuration saved successfully! Please restart the chat service for these changes to take effect."_. So though to avoid restarting but if restarting the app service is already a step for the user if any changes in configuration, We can remove this cache clear code. @adamdougal